docs/web/user_guide.md

```diff
@@ -637,11 +637,39 @@ filter arguments:
                         only if basename or newname is a run name (on the
                         remote server).
   --detected-at TIMESTAMP
-                        Filter results by detection date. The format of
-                        TIMESTAMP is 'year:month:day:hour:minute:second' (the
-                        "time" part can be omitted, in which case midnight
-                        (00:00:00) is used).
-  --fixed-at TIMESTAMP  Filter results by fix date. The format of TIMESTAMP is
+                        DEPRECATED. Use the '--detected-after' option to
```

Comment:
the detected-after or the detected-before options replace this? for the fixed-at the before option is mentioned.
What was the behavior of this filter? Every report which was detected before a given date or at that given date?
What is the replacement flag option?
`--detected-at 2020-08-17` is the same as `--detected-before 2020-08-17`?

Reply:
There are two cases:
- When the detection status is set only to `Resolved`: the `--detected-at/--fixed-at` options can be replaced with the `--fixed-after/--fixed-before` options.
- When detection status is set to a different value (or not set): the `--detected-at/--fixed-at` options can be replaced with the `--detected-after/--detected-before` options.

I updated the help message and the documentation and I have also mentioned this.